src/components/UserCategoryNav.jsx

```diff
@@ -0,0 +1,46 @@
+import {GridFour, CookingPot, NotePencil, Brain} from "@phosphor-icons/react";
+
+const UserCategoryNav = () => {
+  return (
+    <div className="flex gap-8">
+      <div className="flex  flex-col ">
```

Comment:
```suggestion
      <div className="flex flex-col ">
```
Remove extra spaces in this file